e2e/test-component/scenarios/embedding-sdk/interactive-question.cy.spec.tsx

```diff
@@ -475,4 +475,26 @@ describe("scenarios > embedding-sdk > interactive-question", () => {
       cy.findByRole("button", { name: "Visualize" }).should("be.visible");
     });
   });
+
+  it("should not show any sdk error when showing a question in strict mode", () => {
+    cy.get<string>("@questionId").then((questionId) => {
+      mountSdkContent(<InteractiveQuestion questionId={questionId} />, {
+        strictMode: true,
+      });
+
+      getSdkRoot().within(() => {
+        H.assertElementNeverExists({
+          selector: "[data-testid='sdk-error-container']",
+          rejectionMessage:
+            "sdk errors should not show up when rendering an interactive question in strict mode",
+          pollInterval: 20,
+          timeout: 500,
```

Comment:
I think this isn't ideal. My ideal assertion would be "look for `Product ID` until then also ensure that `[data-testid='sdk-error-container']` doesn't not exist"

This way we only have to wait up to until `Product ID` has appear to ensure this assertion works.

This assertion only wait for a specific period of time, in this case, 500ms which could be longer or shorter than `Product ID` could appear. It's not perfect, but that'll do for now.

Can you see if you could implement something like I've suggested? I've spent like maybe 10-20 minutes but couldn't find anything yet. Maybe you could extend `assertElementNeverExists` instead.

Reply:
This is awesome, thanks! I wish we could use `waitFor` + `getElementBy*` in Cypress and do some parallel commands. It should be a lot cleaner, but anyways, this works.